site/en/community/contribute/code_style.md

```diff
@@ -11,24 +11,21 @@ and use [pylint](https://www.pylint.org/) to check your Python changes.
 
 ### pylint
 
-To install `pylint` and retrieve TensorFlow's custom style definition:
+To install `pylint`:
 
 ```bash
-
 $ pip install pylint
-$ wget -O /tmp/pylintrc https://raw.githubusercontent.com/tensorflow/tensorflow/master/tensorflow/tools/ci_build/pylintrc
-
 ```
 
-To check a file with `pylint`:
+To check a file with `pylint` from the TensorFlow source code root directory:
 
 ```bash
-$ pylint --rcfile=/tmp/pylintrc myfile.py
+$ pylint --rcfile=tensorflow/tools/ci_build/pylintrc tensorflow/python/keras/losses.py
 ```
 
 ### Supported Python versions
 
-TensorFlow supports Python >= 3.5. See the
+TensorFlow supports Python >= 3.6. See the
```

Comment:
@zenogantner Do you think it would also make sense to specify the range? For example:

```suggestion
TensorFlow supports Python 3.6-3.8. See the
```

(This is for edge cases where Python 3.9 is auto-installed by Homebrew 🤷‍♂️.)

Source: https://www.tensorflow.org/install/pip, https://www.tensorflow.org/install/